sound/soc/intel/boards/sof_sdw_common.h

```diff
@@ -79,6 +79,24 @@ enum {
 
 #define SOF_SDW_MAX_DAI_NUM		8
 
+static const struct snd_soc_dapm_widget generic_spk_widgets[] = {
+	SND_SOC_DAPM_SPK("Speaker", NULL),
+};
+
+static const struct snd_kcontrol_new generic_spk_controls[] = {
+	SOC_DAPM_PIN_SWITCH("Speaker"),
+};
+
+static const struct snd_soc_dapm_widget generic_jack_widgets[] = {
+	SND_SOC_DAPM_HP("Headphone", NULL),
+	SND_SOC_DAPM_MIC("Headset Mic", NULL),
+};
+
+static const struct snd_kcontrol_new generic_jack_controls[] = {
+	SOC_DAPM_PIN_SWITCH("Headphone"),
+	SOC_DAPM_PIN_SWITCH("Headset Mic"),
+};
```

Comment:
So, these will be added to data each time the sof_sdw_common.h is included (sure, included and used, but I think 9 times with this patch)

Reply:
> So, these will be added to data each time the sof_sdw_common.h is included (sure, included and used, but I think 9 times with this patch)

Maybe move the stuff to sof_sdw.c?